.github/interop/runner.patch

```diff
@@ -64,7 +64,19 @@ index 9150551..55aaa4a 100644
    "mvfst": {
      "image": "lnicco/mvfst-qns:latest",
      "url": "https://github.com/facebookincubator/mvfst",
-@@ -79,8 +74,13 @@
+@@ -49,11 +44,6 @@
+     "url": "https://quic.nginx.org/",
+     "role": "server"
+   },
+-  "msquic": {
+-    "image": "ghcr.io/microsoft/msquic/qns:main",
+-    "url": "https://github.com/microsoft/msquic",
+-    "role": "both"
+-  },
```

Comment:
I'm assuming this was intentional since they're always failing here. Would it be worth removing `chrome` and `xquic` as well? They haven't been working for some time.

Reply:
Yeah, I removed `msquic` because it was always failing but also it was taking a really long time to run (it was the longest of all the impls). `chrome` and `xquic` at least run relatively quickly. `chrome` was succeeding a while back, I'm not sure what happened. `xquic` does succeed on some tests. I think I would just leave those for now